src/preact-redux.d.ts

```diff
@@ -275,5 +275,5 @@ export interface ProviderProps {
  * Makes the Redux store available to the connect() calls in the component hierarchy below.
  */
 export class Provider extends Component<ProviderProps, {}> {
-    render(props: ProviderProps): VNode
+    render(props?: ProviderProps): VNode
```

Comment:
Maybe return type of `JSX.Element` will be better? As long as preact uses signature for render 
`render(props?:PropsType & ComponentProps<this>, state?:StateType, context?:any):JSX.Element|null`